mail_template_company_logo/__manifest__.py

```diff
@@ -0,0 +1,35 @@
+###############################################################################
+#
+#    SDi Soluciones Digitales
+#    Copyright (C) 2021-Today SDi Soluciones Digitales <www.sdi.es>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+###############################################################################
+{
+    'name': 'Mail Template Company Logo',
+    'summary': 'Use a URL from company to set de mail template logo ',
+    'author': 'Jorge Luis Quinteros, SDi Soluciones Digitales',
```

Comment:
```suggestion
#    SDi Digital Group
#    Copyright (C) 2021-Today SDi Digital Group <www.sdi.es>
#
#    This program is free software: you can redistribute it and/or modify
#    it under the terms of the GNU Affero General Public License as
#    published by the Free Software Foundation, either version 3 of the
#    License, or (at your option) any later version.
#
#    This program is distributed in the hope that it will be useful,
#    but WITHOUT ANY WARRANTY; without even the implied warranty of
#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
#    GNU Affero General Public License for more details.
#
#    You should have received a copy of the GNU Affero General Public License
#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
#
###############################################################################
{
    'name': 'Mail Template Company Logo',
    'summary': 'Use a URL from company to set de mail template logo ',
    'author': 'Jorge Luis Quinteros, SDi Digital Group',
```